magicctapipe/scripts/lst1_magic/IT_container_data_MC_bash_scripts/nsb_level_MC.py

```diff
@@ -0,0 +1,106 @@
+"""
+Bash scripts to run LSTnsb.py on all the LST runs by using parallel jobs
+
+Usage: python nsb_level.py (-c config.yaml)
+"""
+
+import argparse
+import glob
+import logging
+import os
+
+import numpy as np
+import yaml
+
+__all__ = ["bash_scripts"]
+
+logger = logging.getLogger(__name__)
+logger.addHandler(logging.StreamHandler())
+logger.setLevel(logging.INFO)
+
+
+def bash_scripts(run, date, config, source, env_name):
+
+    """Here we create the bash scripts (one per LST run)
+
+    Parameters
+    ----------
+    run : str
+        LST run number
+    date : str
+        LST date
+    config : str
+        Name of the configuration file
+    source : str
+        Target name
+    env_name : str
+        Name of the environment
+    """
+
+    lines = [
+        "#!/bin/sh\n\n",
+        "#SBATCH -p long\n",
+        "#SBATCH -J nsb\n",
+        "#SBATCH -N 1\n\n",
+        "ulimit -l unlimited\n",
+        "ulimit -s unlimited\n",
+        "ulimit -a\n\n",
+        f"time conda run -n  {env_name} LSTnsb_MC -c {config} -i {run} -d {date} > {source}_nsblog_{run}.log 2>&1 \n\n",
```

Comment:
I would add also the date to the name of the log file e.g. `{source}_nsblog_{date}_{run}.log`.

Reply:
done